test/test-framework/deploy/crds/cache.example.com_memcachedrs_crd.yaml

```diff
@@ -9,11 +9,12 @@ spec:
     listKind: MemcachedRSList
     plural: memcachedrs
     singular: memcachedrs
-  scope: Namespaced
+  scope: ""
```

Comment:
Why this change?

Reply:
The CRD is outdated. It should be updated when we did the changes in the release 0.11. Another point that we could do is improve it in order to ensure that the test=frameowork is always updated with the current code changes ( + 1 improvement to the sanity ).

Reply:
Added check a well 👍

Reply:
I'm not sure `scope: ""` is valid. Doesn't it need to be either `Namespaced` or `Cluster`?

Reply:
I tested this. It seems like `scope: ""` works and defaults to `Namespaced`, but that confusing to see in the generated CRD in my opinion.

This can be fixed by adding `// +kubebuilder:resource:scope=Namespaced` to the `MemcachedRS` struct type.

Reply:
But looking at [this](https://github.com/kubernetes-sigs/controller-tools/blob/a653df4cae21d140ed307073ccbfb1d7e245d3a6/pkg/crd/markers/crd.go#L307), it seems like it should output `scope: Namespaced` by default. 🤔

Reply:
Solved.